cmd/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/L0Qqi/To-Do_list/database"
+	"github.com/L0Qqi/To-Do_list/internal/app"
+	"github.com/L0Qqi/To-Do_list/internal/taskhandlers"
+
+	_ "modernc.org/sqlite"
+)
+
+func main() {
+
+	db, err := database.InitializeDatabase()
+	if err != nil {
+		log.Fatalf("Ошибка при инициализации базы данных: %v", err)
+	}
+	defer db.Close()
+
+	app := &app.App{DB: db}
+
+	webDir := "../web"
```

Comment:
Вообще обычно проекты запускают из корня `go run cmd/main.go` так что лучше пути от корня делать
```suggestion
	webDir := "./web"
```